site/src/components/TableRowMenu/TableRowMenu.tsx

```diff
@@ -7,7 +7,7 @@ import { MouseEvent, useState } from "react"
 export interface TableRowMenuProps<TData> {
   data: TData
   menuItems: Array<{
-    label: string
+    label: JSX.Element | string | null
```

Comment:
It could be `ReactNode` type instead of `JSX.Element | string | null`

Reply:
Sweet